chromium_src/net/cookies/cookie_monster.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_NET_COOKIES_COOKIE_MONSTER_H_
+#define BRAVE_CHROMIUM_SRC_NET_COOKIES_COOKIE_MONSTER_H_
+
+#define BRAVE_COOKIE_MONSTER_H                             \
+  std::map<std::string, std::unique_ptr<CookieMonster>>    \
+      ephemeral_cookie_stores_;                            \
+  CookieMonster* GetOrCreateEphemeralCookieStoreForDomain( \
```

Comment:
this method doesn't need to be in the header, you can pass in ephemeral_cookie_stores_ to non-class method

Reply:
:+1:

Reply:
I don't understand why fewer methods is better here, but I'm happy to make this change.

Reply:
This method also uses net_log_ from `CookieMonster` in order to create the ephemeral monster. Since it's using to instance variables, I think it's simpler for it to be a method. Is there a reason why having fewer methods here is better?